packages/botonic-cli/src/commands/train.ts

```diff
@@ -46,20 +56,14 @@ class Tasks {
 
   static getByName(taskName: string): Task {
     if (this.isInvalidTask(taskName)) {
-      throw new Error(this.getInvalidTaskMessage(taskName))
+      throw new InvalidTaskError(taskName, Object.keys(this.tasks))
     }
     return this.tasks[taskName]
   }
 
   private static isInvalidTask(taskName: string): boolean {
```

Comment:
change to isValidTask. try to avoid negative functions/constants/variables. Double negations (!isInvalid) make the code more difficult to understand